src/Config/ConfigPanel.ts

```diff
@@ -87,11 +93,31 @@ export class ConfigPanel {
     webview.html = this._getHtmlForWebview(webview, context);
   }
 
-  private _getHtmlForWebview(webview: vscode.Webview, context: vscode.ExtensionContext) {
-    // TODO Use a nonce to only allow specific scripts to be run
-    // TDOO fill with logics related to html
-    // TODO import html and change html source to fit webview format
+  private getPathToFile(webview: vscode.Webview, extensionUri: vscode.Uri, fileName: string) {
+    return webview.asWebviewUri(vscode.Uri.joinPath(extensionUri, 'media/Config', fileName));
```

Comment:
(optional) can we use `this._extensionUri` instread of `extensionUri` so that we can reduce argument numbers.

Reply:
i think so too.
i will remove argument